### PR TITLE
updated locales and other fixes

### PR DESCRIPTION
**Pull request by snepsnepsnep**
-updated locales
-queue gets sent to the UI on UI init
-console error prevention when the UI isn't displayed
-removal of BeamNG file inclusions that seemingly didn't change / do anything for MP
-removed mainmenu css file inclusion while keeping the same styling as before
-switching camera away from passenger instead of throwing errors when the driver node doesn't exist

### CODE REVIEW

**Comment by Starystars67** [comment]
Hey @snepsnepsnep if you drop a comment on here saying you are ready to have this merged I will merge it for you.

File: ui/modules/mainmenu/mainmenu.js [hunk 1]
```diff
@@ -88,12 +88,9 @@ angular.module('beamng.stuff')
             <div layout="row" layout-align="start center" ng-cloak >
               <img src="/ui/modules/mainmenu/drive/icons/account-multiple.svg" style="padding: 5px" height="22px">
               <div layout="row" style="padding: 5px">
-                <span style="padding-left: 5px;">Players: </span>
+                <span style="padding-left: 5px;">Players: <strong>{{ ::beammpData.players }}</strong> </span>
```
Comment (Starystars67) [inline ui/modules/mainmenu/mainmenu.js:91]: Surely doing it this way removes the styling on the players that I had put in?

**Review by Starystars67 — COMMENTED** [review]
Just the one comment, otherwise nice work :)

File: ui/modules/mainmenu/mainmenu.js (hunk 1 above)
Comment (snepsnepsnep) [reply ui/modules/mainmenu/mainmenu.js:91]: I think I matched the styling pretty closely, this is without the css file included
Screenshot 2021-12-30 142414
and this is the one with the included css
image

I just wanted to minimize the amount of maintenance we have to do each BeamNG update.

Comment (Starystars67) [reply ui/modules/mainmenu/mainmenu.js:91]: Oh interesting. I will get back in my box xD

**Review by Starystars67 — APPROVED** [review]

**Review by Anonymous-275 — APPROVED** [review]

**Pull request merged**